testsuite/Rakefile

```diff
@@ -61,6 +61,7 @@ namespace :utils do
   task :generate_smoke_tests do
     clients = Rake::FileList['features/build_validation/init_clients/*.feature']
     clients.reject! { |client| client.include? 'proxy' }
+    clients.reject! { |client| (client.include? 'buildhost') || (client.include? 'terminal') }
```

Comment:
If I may suggest. I would have a template file for buildhost and another template file for the terminal. So we autogenerate buildhost/terminal features for each supported OS.

Reply:
Yup, we need templating. But let's do that in another PR.